pgtoolkit/ctl.py

```diff
@@ -392,6 +387,13 @@ def controldata(self, datadir: Path | str) -> dict[str, str]:
         ).stdout
         return parse_control_data(r.splitlines())
 
+    @property
+    def version(self) -> int:
```

Comment:
It's not cached actually.

Reply:
I was wondering if it would be a good idea or not if it was cached. Hence the inaccurate commit message.

Reply:
In fact, for the async version it's already quite weird to query a property with `await` because in most common cases, properties are not expected to involve complex computations, especially not I/O. At least, in that case (in contrast with the sync case), this is explicit. So I think it should not be a property at all; rather a method, which itself may be cached. Or we can drop it if we don't use it as you suggest in your other comment.

Reply:
Let's drop it.

PR updated.